pkg/envutil/envutil.go

```diff
@@ -0,0 +1,136 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package envutil
+
+import (
+	"os"
+	"slices"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// DefaultBlockList contains environment variables that should not be propagated by default.
+var DefaultBlockList = []string{
```

Comment:
Since you provide a `GetDefaultBlockList()` function there is no reason the variable is exported:

```suggestion
var defaultBlockList = []string{
```